site/committers.html

```diff
@@ -525,13 +526,15 @@
 </ul>
 
 <p>For me, the consequence of these is that we should backport in the following situations:</p>
+
 <ul>
   <li>Both the bug and the fix are well understood and isolated. Code being modified is well tested.</li>
   <li>The bug being addressed is high priority to the community.</li>
   <li>The backported fix does not vary widely from the master branch fix.</li>
 </ul>
 
 <p>We tend to avoid backports in the converse situations:</p>
+
```

Comment:
Before this change, it was:

```diff


-<p>I think the following facets are important to consider:</p>
-<ul>
-  <li>Backports are an extremely valuable service to the community and should be considered for
-any bug fix.</li>
-  <li>Introducing a new bug in a maintenance release must be avoided at all costs. It over time would
-erode confidence in our release process.</li>
-  <li>Distributions or advanced users can always backport risky patches on their own, if they see fit.</li>
-</ul>
-
-<p>For me, the consequence of these is that we should backport in the following situations:</p>
-<ul>
-  <li>Both the bug and the fix are well understood and isolated. Code being modified is well tested.</li>
-  <li>The bug being addressed is high priority to the community.</li>
-  <li>The backported fix does not vary widely from the master branch fix.</li>
-</ul>
-
-<p>We tend to avoid backports in the converse situations:</p>
-<ul>
-  <li>The bug or fix are not well understood. For instance, it relates to interactions between complex
+<p>I think the following facets are important to consider:
+- Backports are an extremely valuable service to the community and should be considered for
+any bug fix.
+- Introducing a new bug in a maintenance release must be avoided at all costs. It over time would
+erode confidence in our release process.
+- Distributions or advanced users can always backport risky patches on their own, if they see fit.</p>
+
+<p>For me, the consequence of these is that we should backport in the following situations:
+- Both the bug and the fix are well understood and isolated. Code being modified is well tested.
+- The bug being addressed is high priority to the community.
+- The backported fix does not vary widely from the master branch fix.</p>
+
+<p>We tend to avoid backports in the converse situations:
+- The bug or fix are not well understood. For instance, it relates to interactions between complex
 components or third party libraries (e.g. Hadoop libraries). The code is not well tested outside
-of the immediate bug being fixed.</li>
-  <li>The bug is not clearly a high priority for the community.</li>
-  <li>The backported fix is widely different from the master branch fix.</li>
-</ul>
+of the immediate bug being fixed.
+- The bug is not clearly a high priority for the community.
+- The backported fix is widely different from the master branch fix.</p>
```